cmd/occollector/app/builder/builder.go

```diff
@@ -34,6 +34,8 @@ type JaegerReceiverCfg struct {
 	ThriftTChannelPort int `mapstructure:"jaeger-thrift-tchannel-port"`
 	// ThriftHTTPPort is the port that the relay receives on for jaeger thrift http requests
 	ThriftHTTPPort int `mapstructure:"jaeger-thrift-http-port"`
+	// ThriftGRPCPort is the port that the relay receives for jaeger grpc requests
+	ThriftGRPCPort int `mapstructure:"jaeger-thrift-grpc-port"`
```

Comment:
jaeger agents use Protobuf w/ gRPC, not Thrift.

Reply:
Thanks @yurishkuro I'll keep in mind. :)